src/worker.cc

```diff
@@ -1123,9 +1123,18 @@ std::string serialize_error(const valhalla_exception_t& exception, Api& request)
   err_stat->set_value(1);
   err_stat->set_type(count);
 
-  // pbf format output
+  // pbf format output, we only send back the info with errors in it
```

Comment:
yeah so we also have to handle errors properly. in this case i think the best signal to whether or not there is an error is to send back nothing but the error info. it is true that its possible to send back partial results but i thought it was better to send back only the error. again we have the same trick for working around the difference between service requests and direct library calls regarding having the `options` object present after seralization even if it was not requested to be included in the response

Reply:
agreed, partial results are tricky..